cdap-api/src/main/java/co/cask/cdap/api/preview/PreviewLogger.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.preview;
+
+/**
+ * Interface used by the CDAP applications to log the preview data.
+ */
+public interface PreviewLogger {
```

Comment:
Shouldn't be call preview logger. We want to use the same API for preview / debug. We can call this `DebugLogger` or even something more generic.